octobot_commons/tentacles_management/abstract_tentacle.py

```diff
@@ -61,57 +41,8 @@ def get_all_subclasses(cls) -> list:
                 subclasses_list += subclass.get_all_subclasses()
         return subclasses_list
 
-    @classmethod
-    def get_config_folder(cls, config_tentacle_type=None) -> str:
-        return f"{TENTACLES_PATH}" \
-               f"/{cls.get_tentacle_folder()}" \
-               f"/{config_tentacle_type or cls.get_config_tentacle_type()}" \
-               f"/{TENTACLE_CONFIG_FOLDER}"
-
-    @classmethod
-    def get_config_file_name(cls, config_tentacle_type=None) -> str:
-        return f"{cls.get_config_folder(config_tentacle_type)}/{cls.get_name()}{CONFIG_FILE_EXT}"
-
-    @classmethod
-    def get_config_file_schema_name(cls, config_tentacle_type=None) -> str:
-        return f"{cls.get_config_folder(config_tentacle_type)}/{cls.get_name()}_{SCHEMA}{CONFIG_FILE_EXT}"
-
-    @classmethod
-    def get_config_file_error_message(cls, error) -> str:
-        return f"Key Error when computing {cls.get_name()} this probably means an error in " \
-               f"{cls.get_config_file_name()} config file" \
-               f". (error: {error})"
-
-    @classmethod
-    def get_specific_config(cls, raise_exception=True) -> dict:
-        try:
-            return load_config(config_file=cls.get_config_file_name())
-        except Exception as e:
-            if raise_exception:
-                raise e
-        return {}
-
-    @classmethod
-    def get_specific_raw_config(cls, raise_exception=True) -> str:
-        try:
-            with open(cls.get_config_file_name()) as file:
-                return file.read()
-        except Exception as e:
-            if raise_exception:
-                raise e
-        return ""
-
-    @classmethod
-    def get_specific_config_schema(cls, raise_exception=True) -> str:
-        try:
-            with open(cls.get_config_file_schema_name()) as f:
-                return f.read()
-        except Exception as e:
-            if raise_exception:
-                raise e
-        return ""
-
     @classmethod
     # returns DESCRIPTION class attribute, used as documentation
+    #TODO replace with markdown doc
```

Comment:
It's a feature in development in Tentacle Manager ?

Reply:
In web interface yes